### DISCUSSION TITLE
dependencies not getting imported with artillery load testing using lambda

### DISCUSSION THREAD

**Discussion by rakesh-tirupathi** [artilleryio/artillery#2338] [Artillery]
Is importing additional modules in custom js code under config processor supported..? i.e. i have custom java script mentioned in config.processor which requires a package to be downloaded from our artifactory. when i run this using artillery lambda i am getting an error `MODULE NOT FOUND` (The additional module is not getting downloaded and packaged when it is uploaded to lambda). But this is working as expected when i run this in local. Is this functionality supported in artillery lambda run..? It is similar to #2145

**Comment by hassy** [comment]
@rakesh-tirupathi yes that's correct, it's one of the current limitations in AWS Lambda support: https://www.artillery.io/docs/load-testing-at-scale/aws-lambda#unavailable-artillery-features
